src/cycax/cycad/engines/cycax_part_freecad.py

```diff
@@ -418,22 +450,26 @@ def build(self, in_name: Path):
         FreeCADGui.activeDocument().activeView().viewTop()
         FreeCADGui.SendMsgToActiveView("ViewFit")
 
-        filepath = f"{self._base_path}/{name}/{name}"
-        doc.saveCopy(str(f"{filepath}.FCStd"))
-        self.render_to_png(Path(filepath))
-        self.render_to_dxf(doc, Path(filepath))
-        self.render_to_stl(doc, Path(filepath))
+        self.filepath = f"{self._base_path}/{name}/{name}"
```

Comment:
Since self._base_path is Path object. It would be better to:
```
self.filepath = self._base_path / name / name
```
or to get me to stop nagging. `self.filepath = self._base_path / name / f"{name}-freecad"

Then when you want to add a suffix you only have to `filepath = self.filepath.with_suffix(".stl")`

Being lazy I would then just stick the view into the suffix. e.g. `filepath = self.filepath.with_suffix(".{view}.png")` to create a file bla.top.png